code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1406,6 +1406,9 @@ GLOBAL_LIST_EMPTY(roundstart_race_names)
 			damage *= 0.8
 		//END OF CITADEL CHANGES
 
+		if(HAS_TRAIT(user, TRAIT_POWER_ARMOR)) // Being in power armor increases unarmed damage
+			damage *= 1.4
```

Comment:
Combat mode + iron fist + power armor melee buff + mace gloves = instant crit.

Let's not.